src/lib/Microsoft.Health.Fhir.R4.Ingest/Template/CodeValueFhirTemplateProcessor.cs

```diff
@@ -141,5 +152,16 @@ protected static CodeableConcept ResolveCode(string type, IList<FhirCode> codes)
                 .ToList(),
             };
         }
+
+        protected static List<CodeableConcept> ResolveCategory(IList<FhirCodeableConcept> categories)
+        {
+            return categories.Select(category =>
+                new CodeableConcept
+                {
+                    Text = category.Text,
+                    Coding = category.Codes.Select(code => new Coding { System = code.System, Code = code.Code, Display = code.Display })

```

Comment:
Will want to make sure a text only category but null codings works correctly as well.  I believe this code will error if the category codes are null.